### PR TITLE
[music] make all discart available via Player.Art(album.discartN)

### PR DESCRIPTION
**Pull request by the-black-eagle**
## Description
Allow skins to access all the discart(s) for an album via Player.Art(album.discartN) where N is a number between 1 and the number of discs the album contains.

## Motivation and Context
Requested by a team-member here https://forum.kodi.tv/showthread.php?tid=349695&pid=2902815#pid2902815 so that all disc art can be shown for a particular album in the same way the video library can show multiple disc artwork for collections.

## How Has This Been Tested?
Runtime tested by changing a variable to display the wrong disc image (eg Player.Art(album.discart2)) when playing a song from disc 1.

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [X] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [X] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [X] All new and existing tests passed

### CODE REVIEW

**Comment by the-black-eagle** [comment]
@DaveTBlake - Boom, another one !!

**Comment by DaveTBlake** [comment]
Hey speedy, you are back with a working brain.... :) Meanwhile I am filling a few minutes before guests arrive. So some ill-considered comments from me...

By "discart" (a common type of art) you actually mean "disc art" = any of the types of art that a disc could have. Currently we only think about front cover (of sleve) and the round CD image ("discart"), but the discs of a boxed set could be in separate packaging and have other art e.g. spines, booklet pages with images or text etc. This effects all art types, so using my favourite art type "wibble"  Player.Art(album.wibbleN).

I know that the prime art for discs is "discart" but maybe reword to remind the world this is more generic than that. 
Eek doorbell ...

**Comment by the-black-eagle** [comment]
Yada yada !!  The core code doesn't care what you call it, it just puts it into the artmap with album.wibbleN or whatever you've named it.  

And yes, I have booklets and stuff but they are really more album related than disc.  Haven't done much with them in Kodi because it's a big skin change to display a booklet (flipping the pages etc etc).  Spine is pretty easy though and tadb has some in it's available artwork.

Anyway, it'll work with whatever the user calls the art (as long as there is a number at the end) and it'll be accessible through Player.Art(album.usernamedartN).  Some standardization is needed though so that skinners and users are both on the same page with regard to naming their art.  If you call it wibbleN then it'll appear in the map but unless a skin is looking for wibbleN then it won't appear on screen.

Enjoy your company.  We had guests once but then I found out I had to give them my alcohol so I don't invite anyone now !!!

**Comment by the-black-eagle** [comment]
Actually, now I've done some more testing, I'm not even sure that this is needed.  I can already access all the art via `ListItem.Art(wibbleN)` or `Player.Art(wibbleN)` so I'm wondering now if it even needs `album.` in front of it ?

I had a play with the album info dialog and I can successfully show all the disc art relating to an album. The screenshot below shows all 3 discs for my SRV set and the first 3 images of the booklet (accessed through `ListItem.Art(bookN)`

<a href="https://ibb.co/w7LcnVm">screenshot048</a>
